### DISCUSSION TITLE
Copied ~/.config/copyq from old machine to new one, see tabs but content is all "<EMPTY>"

### DISCUSSION THREAD

**Discussion by cpsyctc** [hluk/CopyQ#1926] [Q&A]
I hope that subject line is clear.  My old laptop has died, I have fired up a new one, both running Ubuntu 20.04 LTS up to date to today.  I copied ~/.config/copyq from my backup to the new machine and when I start copyq it now shows me all the tabs I had in copyq on the old machine but all the entries just show as "<EMPTY>".  The file ownership is correct and the dat files seem to have non-zero size (below).  Any suggestions? 

Tangentially: I _love_ copyq: saved me days of my life over the last many years and hardly ever gives me problems.

TIA,

Chris

root@Aether1:/home/chris/.config# ls -lsart copyq
total 3060
   4 drwxrwxr-x  2 chris chris    4096 mars   6 13:47  themes
   4 -rw-rw-r--  1 chris chris     116 mars   6 13:47 'copyq_tab_V2ViX2xpbmtzLzg3JQ==.dat'
   4 -rw-rw-r--  1 chris chris     470 mars   6 13:47  copyq_tab_V2ViX2xpbmtzL3pfb3RoZXJz.dat
   4 -rw-rw-r--  1 chris chris     270 mars   6 13:47 'copyq_tab_V2ViX2xpbmtzL3BzeWN0Yw==.dat'
   4 -rw-rw-r--  1 chris chris    2231 mars   6 13:47  copyq_tab_V2ViX2xpbmtzL3BlbGVyaW5hZ2UyMDE2.dat
   4 -rw-rw-r--  1 chris chris     128 mars   6 13:47 'copyq_tab_V2ViX2xpbmtzL2NvbnRhY3RfbWU=.dat'
   4 -rw-rw-r--  1 chris chris     522 mars   6 13:47 'copyq_tab_V2ViX2xpbmtzL1JibG9nL2dlZWtfc3R1ZmY=.dat'
   4 -rw-rw-r--  1 chris chris     117 mars   6 13:47  copyq_tab_V2ViX2xpbmtzL1JibG9n.dat
   4 -rw-rw-r--  1 chris chris    2336 mars   6 13:47 'copyq_tab_V2ViX2xpbmtzL0NPUkU=.dat'
   4 -rw-rw-r--  1 chris chris     123 mars   6 13:47 'copyq_tab_V2ViX2xpbmtzL0NFQ1BmdW5zL2dlZWtfc3R1ZmYvY29kZWNvdg==.dat'
   4 -rw-rw-r--  1 chris chris     908 mars   6 13:47 'copyq_tab_V2ViX2xpbmtzL0NFQ1BmdW5zL2dlZWtfc3R1ZmY=.dat'
   4 -rw-rw-r--  1 chris chris    2328 mars   6 13:47  copyq_tab_V2ViX2xpbmtzL0NFQ1BmdW5z.dat
   4 -rw-rw-r--  1 chris chris    1614 mars   6 13:47  copyq_tab_V2ViX2xpbmtz.dat
   4 -rw-rw-r--  1 chris chris    2414 mars   6 13:47 'copyq_tab_UXVpY2tfcmVzcG9uc2VzL3pPdGhlcg==.dat'
  12 -rw-rw-r--  1 chris chris    9927 mars   6 13:47  copyq_tab_UXVpY2tfcmVzcG9uc2VzL3BlZXJfcmV2aWV3aW5n.dat
  12 -rw-rw-r--  1 chris chris    9590 mars   6 13:47 'copyq_tab_UXVpY2tfcmVzcG9uc2VzL1Jlc2VhcmNoR2F0ZV9SZXF1ZXN0cw==.dat'
   4 -rw-rw-r--  1 chris chris     548 mars   6 13:47 'copyq_tab_UXVpY2tfcmVzcG9uc2VzL1BTWUNITE9QUw==.dat'
 108 -rw-rw-r--  1 chris chris  107315 mars   6 13:47  copyq_tab_UXVpY2tfcmVzcG9uc2VzL0RhbW4gT01ib29r.dat
   4 -rw-rw-r--  1 chris chris     549 mars   6 13:47 'copyq_tab_UXVpY2tfcmVzcG9uc2VzL0NQUl9wYXBlcg==.dat'
   4 -rw-rw-r--  1 chris chris     676 mars   6 13:47 'copyq_tab_UXVpY2tfcmVzcG9uc2VzL0NPUkUvWVAtQ09SRQ==.dat'
   4 -rw-rw-r--  1 chris chris    3444 mars   6 13:47  copyq_tab_UXVpY2tfcmVzcG9uc2VzL0NPUkUvR29hbF9BdHRhaW5tZW50.dat
  16 -rw-rw-r--  1 chris chris   12608 mars   6 13:47  copyq_tab_UXVpY2tfcmVzcG9uc2VzL0NPUkUvc2l0ZV9hbmRfYW5hbHl0aWNz.dat
   4 -rw-rw-r--  1 chris chris     517 mars   6 13:47  copyq_tab_UXVpY2tfcmVzcG9uc2VzL0NPUkUtT01fZm9yX0xBX3BhcGVy.dat
  72 -rw-rw-r--  1 chris chris   71057 mars   6 13:47 'copyq_tab_UXVpY2tfcmVzcG9uc2VzL0NPUkU=.dat'
   4 -rw-rw-r--  1 chris chris    3517 mars   6 13:47  copyq_tab_UXVpY2tfcmVzcG9uc2VzL0lUQU1JVEVE.dat
   8 -rw-rw-r--  1 chris chris    6215 mars   6 13:47 'copyq_tab_UXVpY2tfcmVzcG9uc2VzL0JTUQ==.dat'
   4 -rw-rw-r--  1 chris chris    1088 mars   6 13:47  copyq_tab_UXVpY2tfcmVzcG9uc2Vz.dat
   4 -rw-rw-r--  1 chris chris    1900 mars   6 13:47 'copyq_tab_UGhvbmVfbnVtYmVycw==.dat'
   4 -rw-rw-r--  1 chris chris       4 mars   6 13:47 'copyq_tab_U2NyZWVuc2hvdHM=.dat'
   8 -rw-rw-r--  1 chris chris    6793 mars   6 13:47 'copyq_tab_U2lnbmF0dXJlcw==.dat'
   4 -rw-rw-r--  1 chris chris    2148 mars   6 13:47 'copyq_tab_Tm90ZXNfYW5kX3NuaXBwZXRzL3Bvd2VyZnVsIHF1b3Rlcw==.dat'
   4 -rw-rw-r--  1 chris chris    2019 mars   6 13:47  copyq_tab_Tm90ZXNfYW5kX3NuaXBwZXRzL0NFQ1BmdW5z.dat
   4 -rw-rw-r--  1 chris chris    3633 mars   6 13:47 'copyq_tab_Tm90ZXNfYW5kX3NuaXBwZXRzL0lUX2JpdHMvUg==.dat'
  28 -rw-rw-r--  1 chris chris   24648 mars   6 13:47 'copyq_tab_Tm90ZXNfYW5kX3NuaXBwZXRzL0lUX2JpdHMvTGludXg=.dat'
   4 -rw-rw-r--  1 chris chris    1794 mars   6 13:47 'copyq_tab_Tm90ZXNfYW5kX3NuaXBwZXRzL0lUX2JpdHM=.dat'
   4 -rw-rw-r--  1 chris chris     736 mars   6 13:47  copyq_tab_Tm90ZXNfYW5kX3NuaXBwZXRz.dat
   4 -rw-rw-r--  1 chris chris     596 mars   6 13:47 'copyq_tab_Sm8tYW5uZQ==.dat'
  12 -rw-rw-r--  1 chris chris    8763 mars   6 13:47  copyq_tab_RnJhbmNl.dat
   4 -rw-rw-r--  1 chris chris       4 mars   6 13:47 'copyq_tab_QWRkcmVzc2VzL0pvLWFubmU=.dat'
   8 -rw-rw-r--  1 chris chris    4276 mars   6 13:47  copyq_tab_QWRkcmVzc2Vz.dat
   4 -rw-rw-r--  1 chris chris    3604 mars   6 13:47 'copyq_tab_QmxvZ19wb3N0cy9wZWxlcmluYWdlMjAxNg==.dat'
   4 -rw-rw-r--  1 chris chris     136 mars   6 13:47 'copyq_tab_QmxvZ19wb3N0cy9wc3ljdGM=.dat'
   4 -rw-rw-r--  1 chris chris     136 mars   6 13:47  copyq_tab_QmxvZ19wb3N0cy9DT1JF.dat
   4 -rw-rw-r--  1 chris chris     136 mars   6 13:47 'copyq_tab_QmxvZ19wb3N0cw==.dat'
 596 -rw-rw-r--  1 chris chris  606549 mars   6 13:47 'copyq_tab_JmNsaXBib2FyZA==.dat'
  40 -rw-rw-r--  1 chris chris   39067 mars   6 13:47 'copyq_tab_JmFjY2VudHNfYW5kX3N5bWJvbHM=.dat'
   4 -rw-rw-r--  1 chris chris     302 mars   6 13:47  copyq_tab_d29yay9VRExB.dat
   4 -rw-rw-r--  1 chris chris    1053 mars   6 13:47 'copyq_tab_d29yay9Vb1I=.dat'
   4 -rw-rw-r--  1 chris chris     166 mars   6 13:47 'copyq_tab_d29yay9DbGFyYS9FbGVwaGFudA==.dat'
   8 -rw-rw-r--  1 chris chris    5579 mars   6 13:47 'copyq_tab_d29yay9DbGFyYQ==.dat'
   4 -rw-rw-r--  1 chris chris       4 mars   6 13:47 'copyq_tab_d29yaw==.dat'
1872 -rw-rw-r--  1 chris chris 1913951 mars   6 13:47 'copyq_tab_aW5jYW50YXRpb25zLyZjbGlwYm9hcmQ=.dat'
   4 -rw-rw-r--  1 chris chris     322 mars   6 13:47  copyq_tab_aW5jYW50YXRpb25zL3NoaW55UHN5Y3Rj.dat
  16 -rw-rw-r--  1 chris chris   15333 mars   6 13:47  copyq_tab_aW5jYW50YXRpb25zL0VtaWx5.dat
   4 -rw-rw-r--  1 chris chris    3233 mars   6 13:47  copyq_tab_aW5jYW50YXRpb25zL0NsYXJh.dat
   4 -rw-rw-r--  1 chris chris     916 mars   6 13:47  copyq_tab_aW5jYW50YXRpb25zL0JyaXRsaW5l.dat
   4 -rw-rw-r--  1 chris chris     888 mars   6 13:47  copyq_tab_aW5jYW50YXRpb25zL01CX1JzdHVkaW8tc2VydmVy.dat
   4 -rw-rw-r--  1 chris chris     318 mars   6 13:47  copyq_tab_aW5jYW50YXRpb25zL01CLXBp.dat
   4 -rw-rw-r--  1 chris chris    2734 mars   6 13:47  copyq_tab_aW5jYW50YXRpb25z.dat
   4 -rw-rw-r--  1 chris chris      32 mars   6 13:47  copyq.pub
   4 -rw-rw-r--  1 chris chris    3302 mars   6 13:47  copyq_geometry.ini
  12 -rw-rw-r--  1 chris chris   11657 mars   6 13:47  copyq.conf
   4 -rw-rw-r--  1 chris chris    1877 mars   6 13:47  copyq-commands.ini.bak
   4 -rw-rw-r--  1 chris chris    1877 mars   6 13:47  copyq-commands.ini
  12 -rw-rw-r--  1 chris chris   11657 mars   6 13:47  copyq-bak.conf
   4 -rw-rw-r--  1 chris chris      90 mars   6 19:49  copyq.lock
   0 srwxrwxr-x  1 chris chris       0 mars   6 19:49  .copyq_s
   4 -rw-rw-r--  1 chris chris      36 mars   6 19:49  copyq-filter.ini
   4 -rw-rw-r--  1 chris chris     298 mars   6 19:49  copyq_no_session.ini
   8 -rw-rw-r--  1 chris chris    6036 mars   6 19:50  copyq_tabs.ini
   4 drwxrwxr-x  3 chris chris    4096 mars   6 19:50  .
   4 drwx------ 28 chris chris    4096 mars   6 19:50  ..

**Comment by hluk** [comment]
Any error/warning in the logs? (You can list the logs using `copyq logs` or F12 from the main window.)

**Comment by cpsyctc** [comment]
Ah. I should have thought of that. I have a feeling that I was probably just being impatient because I went back to the old 
machine and exported everything then imported that into the new set up without rechecking and then I found that there 
seemed to be duplicates of everything! I cleaned that all up and probably gained a cleaner collection as a result (there's 
never time for pruning usually!) 

Sorry for the noise and thanks again for CopyQ: brilliant software. 

Chris 

> From: "Lukas Holecek" ***@***.***>
> To: "hluk/CopyQ" ***@***.***>
> Cc: "Chris Evans" ***@***.***>, "Author" ***@***.***>
> Sent: Monday, 7 March, 2022 14:09:26
> Subject: Re: [hluk/CopyQ] Copied ~/.config/copyq from old machine to new one,
> see tabs but content is all "<EMPTY>" (Discussion #1926)

> Any error/warning in the logs? (You can list the logs using copyq logs or F12
> from the main window.)

> —
> Reply to this email directly, [
> https://github.com/hluk/CopyQ/discussions/1926#discussioncomment-2308493 | view
> it on GitHub ] , or [
> https://github.com/notifications/unsubscribe-auth/ACP3COQIM5FFT4EREZK4VFTU6X5YNANCNFSM5QBRNNHQ
> | unsubscribe ] .
> Triage notifications on the go with GitHub Mobile for [
> https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675
> | iOS ] or [
> https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub
> | Android ] .
> You are receiving this because you authored the thread. Message ID:
> ***@***.***>

-- 
Chris Evans (he/him) ***@***.***> 
Visiting Professor, [ https://www.udla.edu.ec/ | UDLA, Quito, Ecuador ] & Honorary Professor, [ https://www.roehampton.ac.uk/ | University of Roehampton ] , London, UK. 
Work web site: [ https://www.psyctc.org/psyctc/ | https://www.psyctc.org/psyctc/ ] 
CORE site: [ http://www.coresystemtrust.org.uk/ | http://www.coresystemtrust.org.uk/ ] 
Personal site: [ https://www.psyctc.org/pelerinage2016/ | https://www.psyctc.org/pelerinage2016/ ] 
Emeetings (Thursdays): [ https://www.psyctc.org/pelerinage2016/ceworkdiary/ | https://www.psyctc.org/pelerinage2016/ceworkdiary/ ] (Beware: French time, generally an hour ahead of UK) 
[ https://uk.sagepub.com/en-gb/eur/outcome-measures-and-evaluation-in-counselling-and-psychotherapy/book244114 ]
